ui/component/viewers/videoViewer/view.jsx

```diff
@@ -167,6 +185,8 @@ function VideoViewer(props: Props) {
       }
 
       setIsLoading(shouldPlay); // if we are here outside of an embed, we're playing
+
+      player.on('loadedmetadata', () => restoreSavedSettings(player));
```

Comment:
In theory, using `ready` should work fine here (or `player.ready(() => restoreSavedSettings(player))`).

I'd prefer to set these values in `ready` if possible. I'm not sure if playback rate can be set in ready though, and may actually require this event handler regardless.

Reply:
> _Do you have a reference for this? I'm curious as to why this would be the case. In my experience restoring state in ready worked best and solved issues like videos autoplaying at max volume for a brief moment before state is restored.
It may be even better to restore state both in ready and loadedmetadata though._

I was referencing this: [github.com/videojs/ video.js/issues/5128] along with my local flow logs.
_Note: I broke up the URL with a space just so that github doesn't link this issue to their repo (reduce "mentioned" noise)_

I tried moving the restoration to after `player.play()` (i.e. at the bottom of `onPlayReady`, or equivalently, at `player.on('ready', ...)`, but it still got overridden. `loadedmetadata` is the earliest event I could find that happen after the reset.

Also tried the idea of leaving the original restoration there, _plus_ adding a redundant one at `loadedmetadata`. Because the connection to LBRY server in my region is slow (i.e. loading screen is longer), I was still able to see the reset, so I end up removing the original restoration.

Reply:
> If you're setting the rate before a source is loaded, use this.player.defaultPlaybackRate(savedRate)

Also tried this given that we update `player.src` in another effect, but didn't seem to work.